File: codewars/js/get_planet_name_by_id.js
// https://www.codewars.com/kata/515e188a311df01cba000003/train/javascript

function getPlanetName(id){
    let planets = {
        1: "Mercury",
        2: "Venus",
        3: "Earth",
        4: "Mars",
        5: "Jupiter",
        6: "Saturn",
        7: "Uranus",
        8: "Neptune",
    };

    return planets[id]
}

console.log(getPlanetName(2))

File: codewars/python/get_planet_name_by_id.py
# https://www.codewars.com/kata/515e188a311df01cba000003/train/python


def get_planet_name(id):
    return [
        "Mercury",
        "Venus",
        "Earth",
        "Mars",
        "Jupiter",
        "Saturn",
        "Uranus",
        "Neptune",
    ][id - 1]


if __name__ == "__main__":
    print(get_planet_name(2))
